test_regress/t/t_hier_block_perf.py

```diff
@@ -14,22 +14,27 @@
 test.cycles = (int(test.benchmark) if test.benchmark else 1000000)
 test.sim_time = test.cycles * 10 + 1000
 
-THREADS = int(os.environ["THREADS"]) if "THREADS" in os.environ else 2
+THREADS = int(os.environ["THREADS"]) if "THREADS" in os.environ else 4
```

Comment:
This will perform badly on Github Actions, where there's only 4 core threads, and we're running multiple tests at once. We were already oversubscribed with 2.

Reply:
Understood, this test describes minimal scenario where hierarchical thread schedules are used; one for parent block and another one for hierarchical children. Without that test, the only way to check this would be via unit tests, which would only validate `PackThreads::pack` algorithm.

Can we run this test sequentially with other multi-threaded tests? 
Apart from this one, we have https://github.com/verilator/verilator/blob/master/test_regress/t/t_hier_block.py which takes 6 threads so we can run it sequentially as well.

Reply:
Ok, I'll watch how long it takes as-is, and add sequential capability if it starts adding up.